apiClient/client.go

```diff
@@ -283,8 +284,31 @@ func (client *Client) doRequest(query string, vars map[string]interface{}, respo
 
 	// run it and capture the response
 	if err := client.Graphql.Run(ctx, req, &responseData); err != nil {
-		err = BuildHttpErrorMessage(err)
+		err = errorsHandler.BuildErrorMessage(err)
 		return err
 	}
 	return nil
 }
+
+func (client *Client) handleCreateError(err error, input map[string]interface{}, resourceType string ) error {
+	parent := input["parent"]
+	if errorsHandler.NotFoundError(err) {
```

Comment:
should this say `parent resource xxxxx not found`

Reply:
sorry actually to be consistent it should be 
`return errors.New(fmt.Sprintf("error reading %s: parent resource not found: %s",resourceType,resource))`